docs/content/docs/dev/table/sql/describe.md

```diff
@@ -112,11 +112,11 @@ table_env = TableEnvironment.create(...)
 # register a table named "Orders"
 table_env.execute_sql( \
         "CREATE TABLE Orders (" 
-        " `user` BIGINT NOT NULl," 
+        " `user` BIGINT NOT NULl comment 'this is primary key'," 
         " product VARCHAR(32),"
         " amount INT,"
-        " ts TIMESTAMP(3),"
-        " ptime AS PROCTIME(),"
+        " ts TIMESTAMP(3) comment 'notice: watermark',"
+        " ptime AS PROCTIME() comment 'this is a computed column',"
```

Comment:
There is not much meaning to adding comments in the input SQL. This is not how to educate users on using COMMENT in CREATE TABLE, but it should show what the SHOW CREATE TABLE displays. Could you add examples and outputs for `SHOW CREATE TABLE` in [this page](https://nightlies.apache.org/flink/flink-docs-master/docs/dev/table/sql/show/#show-create-table)?

Reply:
It is really neccessary to give an example in the doc, done :)